docs/croissant-rai-spec.md

```diff
@@ -0,0 +1,774 @@
+   </td>
+  </tr>
+  <tr>
+   <td>rai:dataReleaseMaintenancePlan
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>Compliance
+   </td>
+   <td>MANY
+   </td>
+   <td>Versioning information in terms of the updating timeframe, the maintainers, and the deprecation policies. 
+   </td>
+  </tr>
+  <tr>
+   <td>rai:personalSensitiveInformation
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>Compliance
+   </td>
+   <td>MANY
+   </td>
+   <td>Sensitive Human Attribute(s)- Gender, Socio-economic status,Geography, Language, Age,Culture, Experience or Seniority, Others (please specify)
+   </td>
+  </tr>
+  <tr>
+   <td>rai:dataSocialImpact
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>AI safety and fairness evaluation
+   </td>
+   <td>ONE
+   </td>
+   <td>Discussion of social impact, if applicable
+   </td>
+  </tr>
+  <tr>
+   <td>rai:dataBiases
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>AI safety and fairness evaluation
+   </td>
+   <td>MANY
+   </td>
+   <td>Description of biases in dataset, if applicable 
+   </td>
+  </tr>
+  <tr>
+   <td>rai:dataLimitations
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>AI safety and fairness evaluation
+   </td>
+   <td>MANY
+   </td>
+   <td>Known limitations - Data generalization limits (e.g related to data distribution, data quality issues, or data sources) and on-recommended uses.
+   </td>
+  </tr>
+  <tr>
+   <td>rai:dataUseCases
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>AI safety and fairness evaluation
+   </td>
+   <td>MANY
+   </td>
+   <td>Dataset Use(s) - Training, Testing, Validation, Development or Production Use, Fine Tuning, Others (please specify), Usage Guidelines. Recommended uses.
+   </td>
+  </tr>
+  <tr>
+   <td>rai:annotationsPerItem
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>Data labeling
+   </td>
+   <td>ONE
+   </td>
+   <td>Number of human labels per dataset item
+   </td>
+  </tr>
+  <tr>
+   <td>rai:annotatorDemographics
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>Data labeling
+   </td>
+   <td>MANY
+   </td>
+   <td>List of demographics specifications about the annotators
+   </td>
+  </tr>
+  <tr>
+   <td>rai:machineAnnotationTools
+   </td>
+   <td><a href="https://schema.org/Text">sc:Text</a>
+   </td>
+   <td>Data labeling
+   </td>
+   <td>MANY
+   </td>
+   <td>List of software used for data annotation ( e.g. concept extraction, NER, and additional characteristics of the tools used for annotation to allow for replication or extension) 
+   </td>
+  </tr>
+</table>
+
+
+
+# Examples
```

Comment:
It would be nice to have those examples as JSON files in https://github.com/mlcommons/croissant/tree/main/datasets/1.0

Reply:
Ack. I believe @JoanGi was planning to add them.

Reply:
Two of them are in this PR: https://github.com/mlcommons/croissant/pull/581 take a look if you can. :)